CefSharp.BrowserSubprocess/Program.cs

```diff
@@ -41,8 +40,29 @@ public static int Main(string[] args)
             }
 
             Debug.WriteLine("BrowserSubprocess shutting down.");
-
+            
             return result;
         }
+
+        private static void AwaitParentProcessExit(int parentProcessId) 
+        {
+            try 
+            {
+                var parentProcess = Process.GetProcessById(parentProcessId);
+                parentProcess.WaitForExit();
+            }
+            catch (Exception e) 
+            {
+                //main process probably died already, bailout
+                Debug.WriteLine(e);
+                return;
```

Comment:
What's the purpose of this return?

Reply:
I assumed that the process would die by it self, but I'll remove it... its safer.